### ISSUE TITLE
Can 'RTCPeerConnectionFactoryOptions' reinterpret to 'mediasoupclient::PeerConnection::Options' ?

### ISSUE THREAD

**Issue by ploverlake** [ethand91/mediasoup-ios-client#80]
Hi, 
I found this statement `mediasoupclient::PeerConnection::Options* pcOptions = reinterpret_cast<mediasoupclient::PeerConnection::Options *>(options);` in the method DeviceWrapper::nativeCreateSendTransport, and the original type of `options` is RTCPeerConnectionFactoryOptions.

**Comment by supermario11** [comment]
@ploverlake Have you solved the problem? I also need to use Options

**Comment by ploverlake** [comment]
> @ploverlake Have you solved the problem? I also need to use Options

I changed the type of param `options` of DeviceWrapper::nativeCreateSendTransport to `void *`, and pass a real argument of `mediasoupclient::PeerConnection::Options *` to this method.

**Comment by supermario11** [comment]
> > @ploverlake Have you solved the problem? I also need to use Options
> 
> I changed the type of param `options` of DeviceWrapper::nativeCreateSendTransport to `void *`, and pass a real argument of `mediasoupclient::PeerConnection::Options *` to this method.

Do I need to compile the framework myself? Can you share it with me? Thank you

**Comment by ploverlake** [comment]
> > > @ploverlake Have you solved the problem? I also need to use Options
> > 
> > 
> > I changed the type of param `options` of DeviceWrapper::nativeCreateSendTransport to `void *`, and pass a real argument of `mediasoupclient::PeerConnection::Options *` to this method.
> 
> Do I need to compile the framework myself? Can you share it with me? Thank you

Yes, I have made a lot of change and compile it, and I am sorry I can not share it with you.

**Comment by supermario11** [comment]
> > > > @ploverlake Have you solved the problem? I also need to use Options
> > > 
> > > 
> > > I changed the type of param `options` of DeviceWrapper::nativeCreateSendTransport to `void *`, and pass a real argument of `mediasoupclient::PeerConnection::Options *` to this method.
> > 
> > 
> > Do I need to compile the framework myself? Can you share it with me? Thank you
> 
> Yes, I have made a lot of change and compile it, and I am sorry I can not share it with you.

OK. Can I have my contact information? Compiler encountered problems to you for help

**Comment by supermario11** [comment]
@ploverlake Friends, can you share your compilation experience? I can't compile it

**Comment by ploverlake** [comment]
> @ploverlake Friends, can you share your compilation experience? I can't compile it

what’s your email？

**Comment by supermario11** [comment]
> > @ploverlake Friends, can you share your compilation experience? I can't compile it
> 
> what’s your email？

657847778@qq.com
